runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -85,20 +131,48 @@ public void addTag(int itemId, String tag)
 		}
 	}
 
+	public void addVariationTag(int itemId, String tag)
+	{
+		final Collection<String> tags = getVariationTags(itemId);
+		if (tags.add(Text.standardize(tag)))
+		{
+			setVariationTags(itemId, tags);
+		}
+	}
+
+	private void setVariationTags(int itemId, Collection<String> tags)
+	{
+		setVariationTagString(itemId, JOINER.join(tags));
+	}
+
 	private void setTags(int itemId, Collection<String> tags)
 	{
 		setTagString(itemId, JOINER.join(tags));
 	}
 
 	boolean findTag(int itemId, String search)
 	{
-		return getTags(itemId).stream().anyMatch(tag -> tag.contains(Text.standardize(search)));
+		Collection<String> tags = getTags(itemId);
+		tags.addAll(getVariationTags(itemId));
+		return tags.stream().anyMatch(tag -> tag.contains(Text.standardize(search)));
 	}
 
 	public void removeTag(String tag)
 	{
 		final String prefix = CONFIG_GROUP + "." + ITEM_KEY_PREFIX;
 		configManager.getConfigurationKeys(prefix).forEach(item -> removeTag(Integer.parseInt(item.replace(prefix, "")), tag));
+
+		final String chargePrefix = CONFIG_GROUP + "." + VARIATION_KEY_PREFIX;
+		configManager.getConfigurationKeys(prefix).forEach(item -> removeVariationTag(Integer.parseInt(item.replace(chargePrefix, "")), tag));
```

Comment:
you can get the configuration keys only once and reuse

Reply:
or, do getConfigurationKeys(chargePrefix) as that looks like correct behaviour here

Reply:
yeah it's a typo